sale_settings/README.rst

```diff
@@ -0,0 +1,67 @@
+
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License: LGPL-3
+
+Sale Settings
+=============
+
+This module add new `company_id` field to the `sale.config.settings` model be
+able to create new related fields to `res.company` model.
+
+This module is created in order to be reused for another modules that want to
+extend the options in the sale configuration wizard.
+
+Installation
+============
+
+To install this module, you need to:
+
+- Not special pre-installation is required, just install as a regular odoo
+  module:
+
+  - Download this module from `Vauxoo/addons-vauxoo
+    <https://github.com/vauxoo/addons-vauxoo>`_
+  - Add the repository folder into your odoo addons-path.
+  - Go to ``Settings > Module list`` search for the current name and click in
+    ``Install`` button.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* There is not special configuration for this module.
+
+Known issues / Roadmap
+======================
+
+Bug Tracker
+===========
+
+Bugs are tracked on
+`GitHub Issues <https://github.com/Vauxoo/addons-vauxoo/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and
+welcomed feedback
+`here <https://github.com/Vauxoo/addons-vauxoo/issues/new?body=module:%20
+sale_settings%0Aversion:%20
+8.0.1.0.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_
```

Comment:
OCA Guidelines mentions that is for the branch the module belongs.. look over https://github.com/OCA/maintainer-tools/issues/151 for more info